src/GraphQL.NewtonsoftJson/StringExtensions.cs

```diff
@@ -31,11 +31,11 @@ public static Inputs ToInputs(this JObject obj)
         }
 
         /// <summary>
-        /// Converts a JSON formatted string into a the dictionary.
+        /// Converts a JSON formatted string into a dictionary.
         /// </summary>
         /// <param name="json">The json.</param>
         /// <returns>Returns a <c>null</c> if the object cannot be converted into a dictionary.</returns>
-        public static Dictionary<string, object> ToDictionary(this string json)
+        private static Dictionary<string, object> ToDictionary(this string json)
```

Comment:
aren't these methods used by the server project?

Reply:
I don't think so anymore. Both deserializers just deserialize straight into `Inputs` which seemed to work, but I think I need to add some more tests around that.

You're right though that if we've exposed them as `public` then potentially we need to keep that going here and in the SystemTextJson project.

Reply:
Have made just the `GetValue` method private for now, will review this again later.